default.nix

```diff
@@ -176,11 +176,41 @@ rec {
         runtimeInputs = with pkgs; [ nix-update ];
         text =
           let
+            skipped-packages = [
+              "atomic-browser" # -> atomic-server
+              "atomic-cli" # -> atomic-server
+              "firefox-meta-press" # -> meta-press
+              "inventaire" # -> inventaire-client
+              "kbin" # -> kbin-backend
+              "kbin-frontend" # -> kbin-backend
+              "pretalxFull" # -> pretalx
+              # FIX: needs custom update script
+              "peertube-plugin-livechat"
+              # FIX: dream2nix
+              "corestore"
+              "liberaforms"
+              # FIX: package scope
+              "bigbluebutton"
+              "heads"
+              # FIX: don't update `sparql-queries` if there is no version change
+              "inventaire-client"
+              # fetcher not supported
+              "libervia-backend"
+              "libervia-desktop-kivy"
+              "libervia-media"
+              "libervia-templates"
+              "sat-tmp"
+              "urwid-satext"
+              # broken package
+              "libresoc-nmigen"
+              "libresoc-verilog"
+            ];
+            update-packages = with lib; filter (x: !elem x skipped-packages) (attrNames ngipkgs);
```

Comment:
We can also add an attribute in the derivations themselves (which will probably become a necessity as the package set increases):

```nix
update-packages =
  with lib;
  attrNames (filterAttrs (_: x: (!x.passthru.skipUpdate or true) || !x.meta.broken or true) ngipkgs);
```

Should we keep it as a list, now, to keep things simple?